### DISCUSSION TITLE
Documentation is a little vague

### DISCUSSION THREAD

**Discussion by Animus-Surge** [pokepetter/ursina#162] [General]
Hello! Apologies if this has already been said, but I've noticed that the documentation in the cheat sheet doesn't go into any detail about what each function does or the gist of each module, and it's been a little tricky for me personally to figure out what does what. I'm curious if it's by design, or if there hasn't been much time to really expand on the documentation much, and if thats the case I'd be more than happy to help get the module documentation in a little more readable state.

**Comment by SinicinDenis** [comment]
this is a good idea! respect
